IsraelHiking.Web/tailwind.config.js

```diff
@@ -2,7 +2,11 @@
 module.exports = {
   content: ['./src/**/*.{html,ts}'],
   theme: {
-    extend: {},
+    extend: {
```

Comment:
Nope. Use "!mt-0" as the class name.

Reply:
Sorry for the confusion revert changes and use !mt-0